slices/dpkg.yaml

```diff
@@ -0,0 +1,32 @@
+package: dpkg
+
+slices:
+  bins:
+    essential:
+      - dpkg_config
+      - libbz2-1.0_libs
+      - libc-bin_ldconfig
+      - libc6_libs
+      - liblzma5_libs
+      - libmd0_libs
+      - libselinux1_libs
+      - libzstd1_libs
+      - tar_tar
+      - zlib1g_libs
+    contents:
+      /usr/bin/dpkg*:
+      /usr/bin/update-alternatives:
+      /usr/libexec/dpkg/*:
+      /usr/sbin/start-stop-daemon:
+      /var/lib/dpkg/tmp.ci/: {make: true}
+      /var/log/dpkg.log: {text: ""}
```

Comment:
Where are these coming from?

Reply:
They came from errors when running the integration tests, but looking again I can't reproduce them if I include the `var` slice I've added, which is from empty directories in the actual tarballs, so I think it's more correct. No idea why `dpkg.log` was a problem before but not now - I'm guessing the error message was bad.